db/migrate/20160710021520_create_tasks.rb

```diff
@@ -0,0 +1,15 @@
+class CreateTasks < ActiveRecord::Migration
+  def change
+    create_table :tasks, id: :uuid do |t|
+      t.string :name, null: false
```

Comment:
I mostly followed the lead of the `Projects` table, but `name` and `state` are nullable there. I made them non-nullable here as we are validating against nulls.